apis/v1alpha1/destination.go

```diff
@@ -0,0 +1,99 @@
+/*
```

Comment:
Generally there's a group name between `apis` and `v1alpha1`...

Reply:
this is a generic component, but not a duck